publishable/routes/web.php

```diff
@@ -49,7 +49,7 @@
         Route::post('request-insurances/{request_insurance}/edit', [
             'uses' => 'RequestInsuranceEditController@create',
             'as'   => 'request-insurance-edits.create',
-        ],)->withoutMiddleware(VerifyCsrfToken::class);
+        ], )->withoutMiddleware(VerifyCsrfToken::class);
```

Comment:
Ikke at det har effekt, men for at holde dette MR lidt mindre så kunne de være fint at revert random mellemrum som disse :smile: